src/main/java/com/rocketden/main/service/GameManagementService.java

```diff
@@ -91,9 +98,28 @@ public RoomDto startGame(String roomId, StartGameRequest request) {
     // Initialize and add a game object from a room object
     public void createAddGameFromRoom(Room room) {
         Game game = GameMapper.fromRoom(room);
+        game.setProblems(getProblemsFromDifficulty(room.getDifficulty(), 1));
         currentGameMap.put(room.getRoomId(), game);
     }
 
+    // Choose the problem based on problem difficulty settings (empty list for invalid request)
+    private List<Problem> getProblemsFromDifficulty(ProblemDifficulty difficulty, int n) {
```

Comment:
Suggestion: could we move this method to the problemService class? Or also now that we have an `n` parameter maybe we could also just modify the original `getRandomProblems` method (since we don't have a need for it anymore)

Reply:
I moved it to the `ProblemService` class 👍  I kept the `getRandomProblems` method because I figured we may want to keep the API call (I'm still open to removing it, but since we already have all the tests and everything configured, I wanted to be more cautious about removing it).